### DISCUSSION TITLE
Material for MkDocs記法サンプル - tsrg.undo

### DISCUSSION THREAD

**Discussion by giscus** [ginteru/For_tsrg.undo#4] [Announcements]
# Material for MkDocs記法サンプル - tsrg.undo

スタイル確認のためにMaterial for MkDocs記法サンプルを記載する

https://tsrg.undo.jp/others/Material_for_MkDocs_sample.html

**Comment by ginteru** [comment]
# これはテスト

テストの本文

## h2レベルの段落

```py
import sys
```
